docs/usage/init.md

```diff
@@ -51,3 +51,21 @@ common:
 ```
 
 Please see <a href="../reference/data_augmentor.html">the data augmentor reference page</a>, when you want to know about augmentation methods. the all of augmentation methods and parameter are explained, methods name in generated yaml correspond to class name under `lmnet.data_augmentor` in the reference.
+
+
+#### optimizer
+
+You can choose optimizer from Adam or Momentum. Each optimizer uses TensorFlow implementation. Please see TensorFlow documentation, [AdamOptimizer](https://www.tensorflow.org/api_docs/python/tf/train/AdamOptimizer) and [MomentumOptimizer](https://www.tensorflow.org/api_docs/python/tf/train/MomentumOptimizer).
```

Comment:
Which optimizer can user use in config yaml, `Adam` or `AdamOptimizer` ? The document looks to use `Adam`, but actually `AdamOptimizer`.
Please change document more clearly what optimizer user can use 👍 

I prefer `Adam` because it is obvious `Adam` is optimizer. But It takes cost to change, no need to change I think 😭

Reply:
`Adam` is 👌